packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import styles from "./componentStyles/Button.module.css";
```

Comment:
Just redeclaring this here so it doesn't get lost in Discord: this approach doesn't work in a standalone React package like it does in Sertra which uses Next.js. We need some tooling to support importing CSS modules into TS.

While this would normally be a blocker, we can allow this now since the components aren't really used anywhere yet. But this assumes we have a task on ClickUp for this, and the work on this issue continues within some near-future milestone.

Reply:
https://app.clickup.com/t/20413243/TS-1230